opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -155,46 +160,47 @@
         SpanBuilder withStartTimestamp(long microseconds);
 
         /**
-         * Returns a newly started and activated {@link ActiveSpan}.
+         * Returns a newly started and activated {@link Scope}.
          *
          * <p>
-         * The returned {@link ActiveSpan} supports try-with-resources. For example:
+         * The returned {@link Scope} supports try-with-resources. For example:
          * <pre><code>
-         *     try (ActiveSpan span = tracer.buildSpan("...").startActive()) {
+         *     try (Scope scope = tracer.buildSpan("...").startActive()) {
          *         // (Do work)
-         *         span.setTag( ... );  // etc, etc
-         *     }  // Span finishes automatically unless deferred via {@link ActiveSpan#capture}
+         *         scope.span().setTag( ... );  // etc, etc
+         *     }  // XXX Span finishes automatically
          * </code></pre>
          *
          * <p>
          * If
          * <ul>
-         * <li>the {@link Tracer}'s {@link ActiveSpanSource#activeSpan()} is not null, and
+         * <li>the {@link Tracer}'s {@link ScopeManager#active()} is not null, and
          * <li>no <b>explicit</b> references are added via {@link SpanBuilder#addReference}, and
          * <li>{@link SpanBuilder#ignoreActiveSpan()} is not invoked,
          * </ul>
          * ... then an inferred {@link References#CHILD_OF} reference is created to the
-         * {@link ActiveSpanSource#activeSpan()}'s {@link SpanContext} when either
+         * {@link ScopeManager#active()}'s {@link SpanContext} when either
          * {@link SpanBuilder#startManual()} or {@link SpanBuilder#startActive} is invoked.
          *
          * <p>
          * Note: {@link SpanBuilder#startActive()} is a shorthand for
-         * {@code tracer.makeActive(spanBuilder.startManual())}.
+         * {@code tracer.scopeManager().activate(spanBuilder.startManual())}.
          *
-         * @return an {@link ActiveSpan}, already registered via the {@link ActiveSpanSource}
+         * @return an {@link Scope}, already registered via the {@link ScopeManager}
          *
-         * @see ActiveSpanSource
-         * @see ActiveSpan
+         * @see ScopeManager
+         * @see Scope
          */
-        ActiveSpan startActive();
+        Scope startActive();
```

Comment:
As with the `ScopeManager.activate`, need to have separate docs and explain above about the default finishOnClose behaviour.